fridgify/lib/data/item_repository.dart

```diff
@@ -38,12 +39,16 @@ class ItemRepository implements Repository<Item> {
 
   @override
   Future<int> add(item) async {
-    // TODO: implement add
-    return null;
+    if(items.containsValue(item)) {
```

Comment:
@DennisRein  Could you elaborate on this implementation ? Setting an id this way seems odd

Reply:
Yeah to be honest, I am not even quiet sure what I did there. This part of the code doesn't even get called anyways. I tried to automatically add non existing items to the item repository on content creation before syncing the repository and I tried to fake a random ID by this implantation. But you are right that's part of the needs to be rethought about/or cleaned away code.

Reply:
So that would be a future refactor then ? Or do you want to change something in this PR ?
I would approve, If you want to do that later.

Reply:
I think we can leave it like that for now and open a refactoring issue for the future. I have a couple things in mind I'll most likely redo to hopefully increase the performance and readability.